community.html

```diff
@@ -65,9 +65,9 @@ <h2><i class="fa fa-comments-o"></i>IRC Channel</h2>
     <hr />
     <p>
       If Discord is not for you, you can also join the
-      <span class="highlight">#scrapy IRC Channel at Freenode</span>
+      <span class="highlight">#scrapy IRC Channel at Libera.Chat</span>
       to chat with other Scrapy users and developers,
-      via <a href="http://webchat.freenode.net/?channels=scrapy">browser</a>
+      via <a href="https://web.libera.chat/">browser</a>
```

Comment:
**suggestion**: `https://web.libera.chat/?channel=#scrapy` fill the Channel field with `#scrapy`

Reply:
Great